src/components/Popover/components/PopoverOverlay/tests/PopoverOverlay.test.tsx

```diff
@@ -143,6 +143,25 @@ describe('<PopoverOverlay />', () => {
     ).toBe('center');
   });
 
+  it('passes preferInputActivator to PositionedOverlay when false', () => {
+    const popoverOverlay = mountWithAppProvider(
+      <PopoverOverlay
+        active
+        id="PopoverOverlay-1"
+        activator={activator}
+        onClose={noop}
+        fixed
+        preferInputActivator={false}
+      >
+        {children}
+      </PopoverOverlay>,
+    );
+
+    expect(
+      popoverOverlay.find(PositionedOverlay).prop('preferInputActivator'),
```

Comment:
Unfortunately, we can't check for the correct `Rect` being passed in the render prop because jsdom will always return `0` for all values. So we thought this is the second best way of assuring that the input has been used.